doc/src/manual/modules.md

```diff
@@ -119,15 +124,11 @@ end
 
 There are three important standard modules: `Main`, `Core`, and `Base`.
 
-`Main` is the top-level module, and Julia starts with `Main` set as the current module.  Variables
-defined at the prompt go in `Main`, and [`varinfo()`](@ref) lists variables in `Main`.
-
-`Core` contains all identifiers considered "built in" to the language, i.e. part of the core language
-and not libraries. Every module implicitly specifies `using Core`, since you can't do anything
-without those definitions.
-
-`Base` is a module that contains basic functionality (the contents of `base/`). All modules implicitly contain `using Base`,
-since this is needed in the vast majority of cases.
+```@docs
+Main
+Core
+Base
+```
```

Comment:
Put these where `Base.Base` was included before, and probably keep the short sentences and just link to the docs.

Reply:
Mhhh `Base.Base` was included in a list of all submodules of `Base`, so `Core` and `Main` don't really fit there.

Reply:
Create a `# Core Modules` section or something in that file then.

Reply:
Okay I moved them.